.build.yml

```diff
@@ -83,6 +101,19 @@ before_scripts:
   - make install
   - cd -
   - rm -Rf libayatana-common-build/
+  -
+  - if [ ${DISTRO_NAME} == "ubuntu" ]; then
+  -     cd ${START_DIR}
+  -     if [ ! -d libxkbcommon-build ]; then
+  -         git clone --depth 1  https://github.com/xkbcommon/libxkbcommon.git libxkbcommon-build
+  -     fi
+  -     cd libxkbcommon-build
+  -     meson setup build -Dprefix=/usr -Dlibdir=/usr/lib/x86_64-linux-gnu
+  -     ninja -C build
+  -     ninja -C build install
+  -     cd -
+  -     rm -Rf libxkbcommon-build/
+  - fi
```

Comment:
I see no reason for building libxkbcommon from source here. Can't we use libxkbcommon in the various versions found in the distros / distro versions that we support?

Reply:
Ubuntu 20.04 does not have `libxkbregistry`.